scripts/daobet_build.sh

```diff
@@ -149,6 +151,12 @@ if [ $# -ne 0 ]; then
          t)
             BUILDTOOLSONLY=1
          ;;
+         f)
+            SKIP_DEPS_CHECK=1
+         ;;
+         n)
```

Comment:
Just in case: `-n` is usually for passing a number or enabling a dry-run mode; `-s` (silent) is more appropriate here, I think.

Reply:
sure, but, unfortunately, `-s` already used for symbol name